blog/entries/models.py

```diff
@@ -7,6 +7,7 @@ class Entry(models.Model):
 	entry_text = models.TextField()
 	entry_date = models.DateTimeField(auto_now_add=True)
 	entry_author = models.ForeignKey(User, on_delete=models.CASCADE)
+	entry_tags = models.ManyToManyField('Tag') # why did Tag need to be a string, or defined before this?
```

Comment:
This string refers to the *name* of the *Model* you want to associate a M2M relationship with

Reply:
I would double check this though in the official docs